cfgov/unprocessed/css/video-player.less

```diff
@@ -57,6 +57,15 @@
     text-align: center;
 }
 
+.video-player_image {
+    opacity: 1;
+    transition: opacity 2s;
```

Comment:
We could explore having unified transitions through the `u-alpha-transition`, `u-alpha-100`, and `u-alpha-0` classes, but it may not be a fit here since the utility transition duration is shorter.

Also, if you need events around the transition, AlphaTransition has them available something like this:

```
var transition = new AlphaTransition( document.querySelector( '.video-player_image' ) );
transition.addEventListener( 'transitionEnd', _handleFadeOut );
transition.fadeTo0(); // NOTE: we should update this to fadeOut/fadeIn
```